docs/api/server/Environment-Variables.md

```diff
@@ -2,6 +2,461 @@
+* ✅ Development
+
+The file path to the public key of an SSL Certificate.
+
+**Shape**
+```bash
+HTTPS_PUBLIC_CERT_CHAIN_PATH=String
+```
+
+**Example**
+```bash
+HTTPS_PUBLIC_CERT_CHAIN_PATH=./some-cert.pem
+```
+
+## `HTTPS_TRUSTED_CA_PATH`
+
+**Runs In**
+* ✅ Production 
+* ✅ Development
+
+The file path to a file containing one or more certs to trust over the system default. See [Node documentation on `ca` option in `tls.createSecureContext`](https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options).
+
+**Shape**
+```bash
+HTTPS_TRUSTED_CA_PATH=String
+```
+
+**Example**
+```bash
+HTTPS_TRUSTED_CA_PATH=./some-extra-certs.pem
+```
+
+## `HTTP_METRICS_PORT`
+
+**Runs In**
+* ✅ Production
+* ✅ Development
+
+Sets the port on which One App's metrics server (e.g. [Prometheus](https://prometheus.io/)) will listen for requests. Defaults to `3005`.
+
+**Shape**
+```bash
+HTTP_METRICS_PORT=Integer
+```
+
+**Example**
+```bash
+HTTP_METRICS_PORT=3005
+```
+
+
+## `HTTP_ONE_APP_DEV_CDN_PORT`
+
+**Runs In**
+* 🚫 Production
+* ✅ Development
+
+Sets the port on which the [One App Dev CDN] server will listen for requests. It defaults to `3001` if `NODE_ENV`
+is `development`, otherwise it is undefined as it is only used for local development.
+
+**Shape**
+```bash
+HTTP_ONE_APP_DEV_CDN_PORT=Integer
+```
+
+**Example**
+```bash
+HTTP_ONE_APP_DEV_CDN_PORT=3001
+```
+
+## `HTTP_ONE_APP_DEV_PROXY_SERVER_PORT`
+
+**Runs In**
+* 🚫 Production
+* ✅ Development
+
+Sets the port on which the [One App Dev Proxy] server will listen for requests. It defaults to `3002` if `NODE_ENV`
+is `development`, otherwise it is undefined as it is only used for local development.
+
+**Shape**
+```bash
+HTTP_ONE_APP_DEV_PROXY_SERVER_PORT=Integer
+```
+
+**Example**
+```bash
+HTTP_ONE_APP_DEV_PROXY_SERVER_PORT=3002
+```
+
+## `HTTP_PORT`
+
+**Runs In**
+* ✅ Production 
+* ✅ Development
+
+Sets the port on which One App will listen for requests. It defaults to `3000` if not defined.
+
+**Shape**
+```bash
+HTTP_ONE_APP_DEV_CDN_PORT=Integer
+```
+
+**Example**
+```bash
+HTTP_PORT=3000
+```
+
+## `IP_ADDRESS`
+
+**Runs In**
+* ✅ Production 
+* ✅ Development
+
+Specify a specific IP Address for One App to bind to.
+
+**Shape**
+```bash
+IP_ADDRESS=String
+```
+
+**Example**
+```bash
+IP_ADDRESS=192.168.1.1
+```
+
+## `NODE_ENV`
+
+**Runs In**
+* ✅ Production 
+* ✅ Development
+
+May be set to either `production` or `development`. When set to `development` additional tooling
+such as `one-app-dev-cdn` and `one-app-dev-proxy` is provided to help with local development. It is undefined
+by default.
+
+Setting this to `development` will set a lot of the other environment variables for you to provide
+sane development time defaults.
+
+Example:
+```bash
+NODE_ENV=production
+```
+
+## `ONE_CLIENT_CDN_URL`
+
+**Runs In**
+* ✅ Production
+* ✅ Development
+
+Fully qualified base path URL where the Module Bundle folders are located (See [Module Map]). Defaults to `/_/static/`.
```

Comment:
Last time I tried this envVar was required in production and didn't default to `/_/static/` was that changed? I couldn't find the PR

Reply:
Fixed in b02bcb1